client/components/common/CandidateModals/ProvideFeedbackModal/index.jsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import BasicModal from '../../BasicModal';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faCheck,
+    faCommentAlt,
+    faTimes
+} from '@fortawesome/free-solid-svg-icons';
+import { Form, FormCheck } from 'react-bootstrap';
+import '../common.css';
+import './ProvideFeedbackModal.css';
+
+const ProvideFeedbackModal = ({
+    at = '',
+    handleAction = () => {},
+    issues = [],
+    show = false,
+    testPlan = '',
+    username = ''
+}) => {
+    const [approveInputDisabled, setApproveInputDisabled] = useState('');
+    const [feedbackInputDisabled, setFeedbackInputDisabled] = useState('');
+    const [changesInputDisabled, setChangesInputDisabled] = useState('');
+
+    const radioChange = element => {
+        switch (element.target.id) {
+            case 'approve-input':
+                setApproveInputDisabled('');
+                setFeedbackInputDisabled('disabled');
+                setChangesInputDisabled('disabled');
+                break;
+            case 'feedback-input':
+                setApproveInputDisabled('disabled');
+                setFeedbackInputDisabled('');
+                setChangesInputDisabled('disabled');
+                break;
+            case 'changes-input':
+                setApproveInputDisabled('disabled');
+                setFeedbackInputDisabled('disabled');
+                setChangesInputDisabled('');
+                break;
+        }
+    };
+
+    return (
+        <BasicModal
+            show={show}
+            actionLabel={'Submit Review'}
+            centered={true}
+            content={
+                <div className="feedback-content">
+                    <p>
```

Comment:
Should this still be shown when 0 issues have been raised? What would it link to?

Reply:
True. @isaacdurazo when 0 issues have been raised, should we remove this text? Remove the link perhaps?

Reply:
Correct, if no issues have been raised, we shouldn't show any text. So the "You have raised X issues for this Test Plan" text should not appear